compiler/testData/diagnostics/tests/modifiers/const/applicability.fir.kt

```diff
@@ -70,20 +70,20 @@ class Outer {
 const val defaultGetter = 19
     <!CONST_VAL_WITH_GETTER!>get<!>
 
-const val nonConstInitializer1 = foo()
-const val nonConstInitializer2 = 1 as String
-const val nonConstInitializer3 = 1.0 as String
-const val nonConstInitializer4 = 1 as Double
-const val nonConstInitializer5 = "2" as Int
-const val nonConstInitializer6 = 1/0
-const val nonConstInitializer7 = -1/0
-const val nonConstInitializer8 = 1/0 - 1/0
-const val nonConstInitializer9 = 1.0/0.0 - 1/0
-const val nonConstInitializer10 = 0/0
-const val nonConstInitializer11 = 1 % 0
-const val nonConstInitializer12 = 0 % 0
-const val nonConstInitializer14 = 0.rem(0)
-const val nonConstInitializer15 = 0.div(0)
+const val nonConstInitializer1 = <!CONST_VAL_WITH_NON_CONST_INITIALIZER!>foo()<!>
+const val nonConstInitializer2 = <!CONST_VAL_WITH_NON_CONST_INITIALIZER!>1 as String<!>
+const val nonConstInitializer3 = <!CONST_VAL_WITH_NON_CONST_INITIALIZER!>1.0 as String<!>
+const val nonConstInitializer4 = <!CONST_VAL_WITH_NON_CONST_INITIALIZER!>1 as Double<!>
+const val nonConstInitializer5 = <!CONST_VAL_WITH_NON_CONST_INITIALIZER!>"2" as Int<!>
+const val nonConstInitializer6 = <!CONST_VAL_WITH_NON_CONST_INITIALIZER!>1/0<!>
+const val nonConstInitializer7 = <!CONST_VAL_WITH_NON_CONST_INITIALIZER!>-1/0<!>
+const val nonConstInitializer8 = <!CONST_VAL_WITH_NON_CONST_INITIALIZER!>1/0 - 1/0<!>
```

Comment:
Why two `DIVISION_BY_ZERO` can't be used here instead of a single `CONST_VAL_WITH_NON_CONST_INITIALIZER`? In my opinion, it looks more clear (the same is also related to FE 1.0).

Reply:
Because `DIVISION_BY_ZERO` is a warning, and `CONST_VAL_WITH_NON_CONST_INITIALIZER` is an error. In this case it's important to have uncompilable code.